src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -392,6 +371,19 @@
       <Link>Common\Unix\System.Native\Interop.GetTimestamp.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' AND '$(IsPartialFacadeAssembly)'!='true' AND '$(TargetGroup)' != 'uap'">
+    <Compile Include="$(CommonPath)\Interop\Windows\shell32\Interop.ShellExecuteExW.cs">
+      <Link>Common\Interop\Windows\shell32\Interop.ShellExecuteExW.cs</Link>
+    </Compile>
+    <Compile Include="System\Diagnostics\Process.Win32.cs" />
+    <Compile Include="System\Diagnostics\ProcessManager.Win32.cs" />
+    <Compile Include="System\Diagnostics\ProcessStartInfo.Win32.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' AND '$(IsPartialFacadeAssembly)'!='true' AND '$(TargetGroup)' == 'uap'">
+    <Compile Include="System\Diagnostics\Process.WinRT.cs" />
```

Comment:
This should be named `*.Uap.cs` according to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md cause it has nothing to do with WinRT.